src/components/map/Map.tsx

```diff
@@ -45,13 +48,27 @@ export default function Map() {
     }
   }
 
+  const getLogit = (percentage: number) => {
```

Comment:
I think it would be a good idea to abstract all this bucketing logic into its own set of functions, doesn't really have much to do with the view layer itself

Reply:
Including this getColors function

Reply:
Making it a todo